### ISSUE TITLE
Focus is not visible

### ISSUE THREAD

**Issue by marc-bouvier** [iroco-co/landing#46]
Quand on utilise le clavier pour naviguer, il n'y a pas d'indicateur (cadre) visible qui indique où se trouve le focus.

https://audits.test.iroco.co/rgaa/4.1/criteres/#10-7

![Screen Recording 2023-11-24 at 17 47 37](https://github.com/iroco-co/landing/assets/15436007/a4bdbc6d-6fc3-4bc2-9eee-2db97c3807ce)

**Comment by marc-bouvier** [comment]
Probablement un fix via un style CSS.

**Comment by Sanket-Arekar** [comment]
I would like to work on this issue. Can you Please assign me this issue?
J'aimerais travailler sur cette question. Pouvez-vous s'il vous plaît m'attribuer ce problème ?

**Comment by marc-bouvier** [comment]
@Sanket-Arekar I am sorry, it seems that @bamthomas had fixed it before we can benefit from your contribution.
